src/main/java/in/projecteka/devservice/common/heartbeat/HeartbeatController.java

```diff
@@ -0,0 +1,26 @@
+package in.projecteka.devservice.common.heartbeat;
+
+import in.projecteka.devservice.common.heartbeat.model.HeartbeatResponse;
+import lombok.AllArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+import reactor.core.publisher.Mono;
+
+import static in.projecteka.devservice.common.Constants.PATH_HEARTBEAT;
+import static in.projecteka.devservice.common.heartbeat.model.Status.UP;
+import static java.time.LocalDateTime.now;
+import static java.time.ZoneOffset.UTC;
+import static org.springframework.http.HttpStatus.OK;
+import static reactor.core.publisher.Mono.just;
+
+@RestController
+@AllArgsConstructor
+public class HeartbeatController {
+
+    @GetMapping(PATH_HEARTBEAT)
+    public Mono<ResponseEntity<HeartbeatResponse>> getProvidersByName() {
```

Comment:
`getProvidersByName` this could be renamed to `heartbeat`

Reply:
Done.